lighthouse/lighthouse-auth.test.ts

```diff
@@ -0,0 +1,274 @@
+import fs from 'fs';
+import { loadInitialPage, main } from './lighthouse-auth.mjs';
+
+describe('Lighthouse Pre-Login Steps', () => {
+  const OLD_ENV = process.env;
+  const previewUrl = 'https://pr-908.d3dytjb8adxkk5.amplifyapp.com';
```

Comment:
The tests will keep working once this preview environment is gone, right? This could be any URL since you mock `initialPage.goto`, right?